vm/class.go

```diff
@@ -1308,6 +1309,22 @@ var builtinClassCommonInstanceMethods = []*BuiltinMethodObject{
 
 		},
 	},
+	{
+		Name: "rand",
+		Fn: func(receiver Object, sourceLine int, t *Thread, args []Object, blockFrame *normalCallFrame) Object {
+			aLen := len(args)
+			switch aLen {
+			case 0:
+				return t.vm.initFloatObject(rand.Float64())
+			case 1:
+				return t.vm.InitIntegerObject(rand.Intn(args[0].Value().(int)))
```

Comment:
we probably should check arguments' types first. I think `.(int)` will make Go panicking when users pass something like a string.